aws/rds.go

```diff
@@ -0,0 +1,95 @@
+package aws
+
+import (
+	"time"
+
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+)
+
+func waitUntilRdsDeleted(svc *rds.RDS, input *rds.DescribeDBInstancesInput) error {
+	for i := 0; i < 240; i++ {
+		_, err := svc.DescribeDBInstances(input)
+		if err != nil {
+			if awsErr, isAwsErr := err.(awserr.Error); isAwsErr && awsErr.Code() == "DBInstanceNotFound" {
+				return nil
+			}
+
+			return err
+		}
+
+		time.Sleep(1 * time.Second)
+		logging.Logger.Debug("Waiting for RDS to be deleted")
+	}
+
+	return RdsDeleteError{}
+}
+
+func getAllRdsInstances(session *session.Session, region string, excludeAfter time.Time) ([]*string, error) {
+	svc := rds.New(session)
+
+	result, err := svc.DescribeDBInstances(&rds.DescribeDBInstancesInput{})
+
+	if err != nil {
+		return nil, errors.WithStackTrace(err)
+	}
+
+	var names []*string
+
+	for _, database := range result.DBInstances {
+		//if excludeAfter.After(*database.InstanceCreateTime) {
+		names = append(names, database.DBInstanceIdentifier)
+		//}
+	}
+
+	return names, nil
+}
+
+func nukeAllRdsInstances(session *session.Session, names []*string) error {
+	svc := rds.New(session)
+
+	if len(names) == 0 {
+		logging.Logger.Infof("No Relational Database Service to nuke in region %s", *session.Config.Region)
```

Comment:
NIT:

```suggestion
		logging.Logger.Infof("No RDS DB Instances to nuke in region %s", *session.Config.Region)
```

RDS (Relational Database Service) is the AWS service, and what we are nuking are the DB instances that are a part of that.

Reply:
Looks like this is in several places, so be sure to update all references as well.

Reply:
Thank you! I wasn't sure how to represent this.